### DISCUSSION TITLE
Some width classes doesn't work

### DISCUSSION THREAD

**Discussion by Puzo** [tailwindlabs/tailwindcss#9852] [Help] [answered]
Tailwindcss version: 3.2.4
Postcss: 8.4.x
Roots/{bud, bud-postcss, bud-sass, bud-tailwindcss, sage}: 6.4.5

I am using Sage starter template from Roots (https://roots.io/sage/) which it comes with Tailwindcss by default - that's great btw.
I noticed, that some of classes are missing, for example w-4/12. For example, my html structure to build grid:

```
<div class="flex">
  <div class="w-4/12">Example 1</div>
  <div class="w-4/12">Example 2</div>
  <div class="w-4/12">Example 3</div>
</div>
```

Parent element with `flex` class is built normaly and classes `w-4/12` are missing.
Also in my generated `.css` file classes aren't there. For example, in my `.css` file there is class `w-4/6`.

My app.scss file:
```
@tailwind base;
@tailwind components;
@tailwind utilities;
@tailwind variants;
```

I tried also to include styles like (same problem):
```
@import 'tailwindcss/base';
@import 'tailwindcss/components';
@import 'tailwindcss/utilities';
@import 'tailwindcss/variants';
```

My `tailwind.config.cjs` file:
```
const defaultTheme = require('tailwindcss/defaultTheme')

module.exports = {
  content: [
    "./index.php", 
    "./app/**/*.php", 
    "./resources/**/*.{php,vue,js}",
    "./resources/**/*.blade.php"
  ],
  theme: {
    container: {
      ...defaultTheme.container,
      center: true
    },
    extend: {
      screens: {
        '2xl': '1280px'
      },
      colors: {},
      aspectRatio: {
        '16/10': '16 / 10',
        '3/4': '3 / 4'
      },
    },
  },
  plugins: [],
};

```

 Any idea why - what am I missing?

**Comment by wongjn** [comment]
Consider checking that the file that the code resides in is sufficiently covered by the `content` globs in the Tailwind configuration.

**Reply by Puzo** [reply]
It is - or can I somehow debug it? I have file for example in folder: `resources/views/modules/test.blade.php` which is covered in glob pattern `./resources/**/*.blade.php`.

**Comment by Puzo** [comment, marked as answer]
I found the problem. It's in the loading dynamic class name - https://tailwindcss.com/docs/content-configuration#classes-aren-t-generated

I am echoing class name dynamic via some helper functions: `<div class="{!! get_sub_field("column_width") !!}">`

My workaround is in that I added safelist (everything) to my `tailwind.config.cjs` file:
```
safelist: [
    {
      pattern: /./,
    }
  ],
```

**Comment by yangtingting123456** [comment]
这是来自QQ邮箱的假期自动回复邮件。你好，我最近正在休假中，无法亲自回复你的邮件。我将在假期结束后，尽快给你回复。
